include/GLKit/GLKMath.h

```diff
@@ -198,36 +195,36 @@ GLKIT_EXPORT GLKMatrix4 GLKMatrix4MakeWithArray(float* values);
 GLKIT_EXPORT GLKMatrix4 GLKMatrix4MakeWithArrayAndTranspose(float* values);
 GLKIT_EXPORT GLKMatrix4 GLKMatrix4MakeWithColumns(GLKVector4 r0, GLKVector4 r1, GLKVector4 r2, GLKVector4 r3);
 GLKIT_EXPORT GLKMatrix4 GLKMatrix4MakeWithRows(GLKVector4 r0, GLKVector4 r1, GLKVector4 r2, GLKVector4 r3);
-GLKIT_EXPORT GLKMatrix4 GLKMatrix4MakeWithQuaternion(GLKQuaternion quaternion) STUB_METHOD;
+GLKIT_EXPORT GLKMatrix4 GLKMatrix4MakeWithQuaternion(GLKQuaternion quaternion);
 GLKIT_EXPORT GLKMatrix4 GLKMatrix4MakeOrthonormalXform(GLKVector3 right, GLKVector3 up, GLKVector3 forward, GLKVector3 pos);
 
 GLKIT_EXPORT GLKMatrix3 GLKMatrix3MakeWithArray(float* values);
 GLKIT_EXPORT GLKMatrix3 GLKMatrix3MakeWithArrayAndTranspose(float* values);
 GLKIT_EXPORT GLKMatrix3 GLKMatrix3MakeWithColumns(GLKVector3 r0, GLKVector3 r1, GLKVector3 r2);
 GLKIT_EXPORT GLKMatrix3 GLKMatrix3MakeWithRows(GLKVector3 r0, GLKVector3 r1, GLKVector3 r2);
-GLKIT_EXPORT GLKMatrix3 GLKMatrix3MakeWithQuaternion(GLKQuaternion quaternion) STUB_METHOD;
+GLKIT_EXPORT GLKMatrix3 GLKMatrix3MakeWithQuaternion(GLKQuaternion quaternion);
 
-GLKIT_EXPORT GLKMatrix3 GLKMatrix3MakeScale(float sx, float sy, float sz) STUB_METHOD;
+GLKIT_EXPORT GLKMatrix3 GLKMatrix3MakeScale(float sx, float sy, float sz);
 
 GLKIT_EXPORT GLKMatrix2 GLKMatrix3GetMatrix2(GLKMatrix3 matrix);
-GLKIT_EXPORT GLKVector3 GLKMatrix3GetColumn(GLKMatrix3 matrix, int column) STUB_METHOD;
-GLKIT_EXPORT GLKVector3 GLKMatrix3GetRow(GLKMatrix3 matrix, int row) STUB_METHOD;
-GLKIT_EXPORT GLKMatrix3 GLKMatrix3SetColumn(GLKMatrix3 matrix, int column, GLKVector3 vector) STUB_METHOD;
-GLKIT_EXPORT GLKMatrix3 GLKMatrix3SetRow(GLKMatrix3 matrix, int row, GLKVector3 vector) STUB_METHOD;
+GLKIT_EXPORT GLKVector3 GLKMatrix3GetColumn(GLKMatrix3 matrix, int column);
+GLKIT_EXPORT GLKVector3 GLKMatrix3GetRow(GLKMatrix3 matrix, int row);
+GLKIT_EXPORT GLKMatrix3 GLKMatrix3SetColumn(GLKMatrix3 matrix, int column, GLKVector3 vector);
+GLKIT_EXPORT GLKMatrix3 GLKMatrix3SetRow(GLKMatrix3 matrix, int row, GLKVector3 vector);
 
 GLKIT_EXPORT GLKMatrix3 GLKMatrix3MakeRotation(float rad, float x, float y, float z);
 GLKIT_EXPORT GLKMatrix3 GLKMatrix3MakeXRotation(float rad);
 GLKIT_EXPORT GLKMatrix3 GLKMatrix3MakeYRotation(float rad);
 GLKIT_EXPORT GLKMatrix3 GLKMatrix3MakeZRotation(float rad);
 
 GLKIT_EXPORT GLKMatrix2 GLKMatrix4GetMatrix2(GLKMatrix4 m);
-GLKIT_EXPORT GLKMatrix3 GLKMatrix3Invert(GLKMatrix3 m, BOOL* isInvertible);
-GLKIT_EXPORT GLKMatrix3 GLKMatrix3InvertAndTranspose(GLKMatrix3 m, BOOL* isInvertible);
+GLKIT_EXPORT GLKMatrix3 GLKMatrix3Invert(GLKMatrix3 m, bool* isInvertible);
```

Comment:
That's unfortunate - its not very useful if your code doesn't compile.  Apologies for sending you down that path.  Looking at the file, it looks like it was already a mix of bool and BOOL and it indeed does compile fine even with some private code we have internally.  So please go ahead and change it back to bool.  I owe you big time.




---
In reply to: [90193755](https://github.com/Microsoft/WinObjC/pull/1370#discussion_r90193755) [](ancestors = 90193755)